From c7c3a01f426089c4023d228d886a3a8bff369298 Mon Sep 17 00:00:00 2001
From: Dmitriy Paliy <dmitriy.paliy@nokia.com>
Date: Thu, 13 Oct 2011 11:09:27 +0300
Subject: [PATCH BlueZ 1/3] Fix crash on power off in maemo6 after sending DTMF

This fixes regression introduced in
b48e610314fb466035b198f594c5b6a68126fcf1. Crash happens when powering
off adapter after sending DTMF tone.
---
 audio/telephony-maemo6.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/audio/telephony-maemo6.c b/audio/telephony-maemo6.c
index 82cb274..1910410 100644
--- a/audio/telephony-maemo6.c
+++ b/audio/telephony-maemo6.c
@@ -771,7 +771,7 @@ static void start_dtmf_reply(DBusPendingCall *call, void *user_data)
 				DBUS_TYPE_INVALID);
 
 	dbus_message_unref(reply);
-	dbus_pending_call_unref(call);
+	remove_pending(call);
 }
 
 static void start_dtmf(void *telephony_device, char tone)
-- 
1.7.4.1

